paddle/framework/error.h

```diff
@@ -14,19 +14,18 @@ limitations under the License. */
 
 #pragma once
 
-#include <glog/logging.h>
```

Comment:
Let's try don't use the string typed error until we know we must use it.  I had this idea because at least in @jacquesqiao 's PR https://github.com/PaddlePaddle/Paddle/pull/2629#discussion_r124430845, we don't need this error type.